wordpress-mobile/WordPress-Android
# [Reader IA] Open subscription management from Filter sheet

Fixes #19859 

> [!CAUTION]
> Should not be merged before #19856

Adds the `Edit` button to the Filter bottom sheet and opens the existing "Subscription Management" screen when tapping that button.

>[!Important]
> In the Reader IA project we are using the word "tag" instead of "topic" and "blog" instead of "site" but the existing subscription management UI uses the previous terminology. We are still figuring out what should be done with respect to that, a conversation started here: p1703879376821999-slack-C05N140C8H5

-----

## To Test:

1. Install and log into Jetpack
2. Go to the Reader
3. Select a Feed that contains filters (like "subscriptions")
4. Tap the "Blog" filter
5. **Verify** the "Edit" button is shown in the sheet (top right corner)
6. Tap the "Edit" button
7. **Verify** it opens the existing subs management screen in the "Sites" page
8. Go back
9. Tap the "Tag" filter
10.  Tap the "Edit" button
11. **Verify** it opens the existing subs management screen in the "Topics" page

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

12. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

13. What automated tests I added (or what prevented me from doing so)

    - Updated existing test naming since an existing flow was used.

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
